src/category_theory/opposites.lean

```diff
@@ -52,6 +53,31 @@ by { rintros _ _ ⟨ ⟩, refl }
 lemma unop_inj : function.injective (@unop C) :=
 by { rintros _ _ ⟨ ⟩, refl }
 
+def op_induction {F : Π (X : Cᵒᵖ), Sort v₁} (h : Π X, F (op X)) : Π X, F X :=
+λ X, h (unop X)
+
+end category_theory
+
+namespace tactic.interactive
+
+open interactive
```

Comment:
Please merge these 4 `open` lines. Or am I missing some reason why they should be on 4 separate lines?

Reply:
Fixed.